### DISCUSSION TITLE
Improve coverlet visibility/reachability/utility

### DISCUSSION THREAD

**Discussion by MarcoRossignoli** [coverlet-coverage/coverlet#1255] [Ideas]
Coverlet is pretty mature tool used by a lot of devs and injected by design into dotnet test template.
It's also officially indicated as open solution for coverage by MS https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-code-coverage

We should improve the "brand" and try to be more supportive, clear, useful adding some new channels for documentation/informations, some idea:

* create a website(we already own coverlet-coverage.com) a static site repo on our github would be enough
* open twitter account to announce release, fix etc...
* create a gitter community to let users talk each other on ideas, solutions
* create a new repo coverlet-docs where to put samples/scenario in a structured way adding also for instance CI/CD yaml sample

cc: @daveMueller @petli @tonerdo

**Comment by Bertk** [comment]
Closing this discussion since it’s no longer relevant to the current state of the project.
